api/src/main/resources/workbench.yaml

```diff
@@ -277,6 +277,25 @@ paths:
           schema:
             $ref: "#/definitions/Profile"
 
+  /v1/account/verify-email:
```

Comment:
High-level question: do we need a separate endpoint for this? Shouldn't we automatically send an e-mail verification request when you create your account?

Reply:
Related – why do we have a user name (which is an email address and stored in the user.email field, both required and unique) and a contact email (which, according to the DB, can be NULL or a duplicate)? If we need both, why not verify non-NULL and unique email?

Reply:
User name is in our Gsuite domain (researchallofus.org in prod), and won't have an e-mail inbox.

contact email is where you actually get e-mail. 

I'd be up for making contact email not null. We should have a discussion about uniqueness. It's possible there is a legitimate reason to make multiple user accounts with a single email address.

Reply:
We don’t currently require a user to put in a contact email at account creation, and this allows users to relatively easily change their contact email (and then reverify). That makes me inclined to leave it as a separate endpoint.

Reply:
IMHO we *should* require an email at account creation, and send out a verification email immediately. This doesn't prevent them from changing their contact email later; if they do that, we should send out another.

This is assuming there's no point in having users have to click a button to say "yes send me an e-mail please"... we should just tell them we did it.